messagebus/src/vespa/messagebus/steadytimer.cpp

```diff
@@ -0,0 +1,15 @@
+// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+#include "steadytimer.h"
+#include <chrono>
+
+using namespace std::chrono;
+
+namespace mbus {
+
+uint64_t
```

Comment:
consider some kind of duration here in the future

Reply:
Added Todo